apps/linode-marketplace-influxdb/roles/influx/templates/docker-compose.yaml

```
@@ -0,0 +1,14 @@
+version: '3'
+services:
+  influxdb:
+    image: influxdb:latest
+    ports:
+      - 8086:8086
+    volumes:
+      - /var/lib/influxdb:/var/lib/influxdb
```

```suggestion
      - /var/lib/influxdb:/var/lib/influxdb2
```
@tbaka data is not persisting because the container mapping is incorrect. This needs to be updated to the suggested to ensure we are reading from the correct path. One that is done, creating new dashboards/notebooks persist when the container is restarted

---

Corrected in [b62ab04](https://github.com/akamai-compute-marketplace/marketplace-apps/pull/120/commits/b62ab040728a1c112d0f5894c307bfd2f1dd3f3c).